rdf-toolkit.md

```diff
@@ -0,0 +1,18 @@
+# How to start:
+  - on windows: drag the ontology file to the "rdf-toolkit.bat" file
+    or start the `cmd.exe` in the current directory
+    run the `rdf-toolkit.bat` script with the ontology file as an argument, e.g.:
+    ```
+    rdf-toolkit.bat ontology.ttl
```

Comment:
not sure about ontology.ttl to the correct our .ttl file to `ontohgis.ttl`

Reply:
Resolved in commit 49fff4a